src/class.js

```diff
@@ -18,7 +29,23 @@
 // Cat should have the property `name` that is set in the constructor and the method
 // `meow` that should return the string `<name> meowed!` where `<name>` is the `name`
 // property set on the Cat instance.
-
+class Animal {
+  constructor(options) {
+    this.age = options.age;
+    this.growOlder = function () {
+      return this.age;
```

Comment:
Should this be `return this.age++;`? I'm just going off the name of the function.